src/main/java/com/khomishchak/cryproportfolio/model/Transaction.java

```diff
@@ -0,0 +1,27 @@
+package com.khomishchak.cryproportfolio.model;
+
+import org.springframework.stereotype.Service;
+
+import java.util.Date;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class Transaction {
+
+    private String transactionId;
+    private String transactionHash;
+    private String ticker;
+    private double fee;
+    private double amount;
```

Comment:
Do not use `double` for money stuff. You can use instead `BigDecimal`.
[here is simple example](https://saturncloud.io/blog/why-not-use-double-or-float-to-represent-currency/)